### PR TITLE
cmd: extend gh and replace bot

### PR DESCRIPTION
**Pull request by whilp**
First step towards getting renovate working again.

### CODE REVIEW

**Comment by statik** [comment]
nice!

**Comment by whilp** [comment]
@statik fwiw, actual fix is:

https://github.com/whilp/world/pull/647

TLDR, *I* broke the workflow when I removed the explicit 'id' on the 'gh' step. I interpreted ambiguity in the docs to mean that the name of a step acted as an implicit id (it clearly does not). This caused renovate to get an empty string as RENOVATE_TOKEN, which then caused renovate to...run out of memory? I got a little insight by setting NODE_OPTIONS=--max-old-space-size=8192 (via https://support.snyk.io/hc/en-us/articles/360002046418-JavaScript-heap-out-of-memory), which avoided the OOM but resulted in similarly obscure error. 

All of which is to say, I think your bot changes might have been innocent! Not the first innocent thing to get zapped, I guess.

**Comment by whilp** [comment]
The last thing I wanna do is shim gh into the renovate image and republish it on my ghcr.io:

https://github.com/whilp/world/pull/646

That should avoid the separate gh step, which would also avoid this issue (and run more quickly).

**Comment by statik** [comment]
> That should avoid the separate gh step, which would also avoid this issue (and run more quickly).

Nice! That will be super clean.
